tests/postgres/query.rs

```diff
@@ -1255,6 +1255,26 @@ fn insert_8() {
     );
 }
 
+#[test]
+fn insert_10() {
+    assert_eq!(
+        Query::insert()
+            .into_table(Glyph::Table)
+            .columns([Glyph::Aspect, Glyph::Tokens])
+            .values_panic([
+                3.1415.into(),
+                vec![
+                    "Token1".to_string(),
+                    "Token2".to_string(),
+                    "Token3".to_string()
+                ]
+                .into()
+            ])
+            .to_string(PostgresQueryBuilder),
+        r#"INSERT INTO "glyph" ("aspect", "tokens") VALUES (3.1415, '{Token1,Token2,Token3}')"#
```

Comment:
Umm I am not sure whether the `Token1` should be quoted at all?
It seems more readable using the `ARRAY` syntax instead?
```sql
INSERT INTO "glyph" ("aspect", "tokens") VALUES (3.1415, ARRAY ['Token1','Token2','Token3'])
```

Reply:
If I understand your first note correctly, I think yes, we should put those strings inside quotes, otherwise, the database would think of them as they're variables or column names and it throws an error like this: 
`ERROR: column "Token1" does not exist`

About using `ARRAY`: I agree, I can make this change as well. Thanks for noting.